pkgs/development/haskell-modules/configuration-nix.nix

```
@@ -818,6 +818,12 @@ self: super: builtins.intersectAttrs super {
     platforms = pkgs.lib.platforms.x86;
   } super.geomancy;
 
+  hlint = overrideCabal (drv: {
+    postInstall = ''
+      install -Dm644 data/hlint.1 -t "$out/share/man"
```

Oh, also completely missed this: needs to go into the `man1` subdirectory or man page viewers won't find it:

```suggestion
      install -Dm644 data/hlint.1 -t "$out/share/man/man1"
```